.github/workflows/release-kics-queries-repo-branch.yaml

```diff
@@ -0,0 +1,46 @@
+name: release-kics-queries-repo-branch
+on:
+  release:
+    types:
+      - published
+jobs:
+  create-branch:
+    runs-on: ubuntu-latest
+    env:
+      REPO_NAME: "kics-queries-repo"
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Git credentials
+      run: |
+          git config --global user.name "${{ github.actor }}"
+          git config --global url."https://${{ secrets.CHECKMARXDEV_GIT_TOKEN }}@github.com".insteadOf "https://github.com"
+
+    - name: Clone kics-queries-repo from CheckmarxDev
+      run: |
+          cd ..
+          git clone https://github.com/CheckmarxDev/$REPO_NAME.git
```

Comment:
Should the use the environment variable?

Reply:
This var was created to avoid the repetition of kics-queries-repo since there are a lot of references to it